Week01/Problem02/bsuttner_02.py

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env python3
+"""Each new term in the Fibonacci sequence is generated by adding the previous two terms. 
+By starting with 1 and 2, the first 10 terms will be:
+1, 2, 3, 5, 8, 13, 21, 34, 55, 89, ...
+By considering the terms in the Fibonacci sequence whose values do not exceed four million, find the sum of the even-valued terms."""
+
+previous, current = 0, 1
+total = 0
+while True:
```

Comment:
Even though the `while True: condition break` structure works, I'd counsel against writing loops this way because if you ever forget to write/have a bug in your `break` statements, you'll get caught in an infinite loop that will crash your program.  Instead, you can write
```
while current <= 4e6:
    ...
```
to achieve the same result as the first three lines of your loop.